puppet/modules/quickstack/manifests/neutron/all.pp

```diff
@@ -73,6 +73,11 @@
   $n1kv_vsm_ip                   = undef,
   $n1kv_vsm_password             = undef,
   $n1kv_vsm_username             = undef,
+  $plumgrid_director_vip         = undef,
```

Comment:
For director VIP,  we ask user to reserve an IP for PLUMgrid director in the static/DHCP pool before the deployment install. I don't think it's going to conflict with pacemaker VIPs since it will be out of that pool.